requirements.txt

```diff
@@ -1,8 +1,7 @@
 trueskill
-quamash
 mock
-PySide
```

Comment:
Indeed PySide doesn't have a binary pip distribution, but it is a requirement, so it should be in here.